moveit_core/kinematics_base/src/kinematics_base.cpp

```diff
@@ -92,6 +92,17 @@ bool KinematicsBase::initialize(const rclcpp::Node::SharedPtr& /*node*/,
   return false;
 }
 
+bool KinematicsBase::initialize(const moveit::core::RobotModel& /*robot_model*/, const std::string& group_name,
+                                const std::string& /*base_frame*/, const std::vector<std::string>& /*tip_frames*/,
+                                double /*search_discretization*/)
+{
+  RCLCPP_ERROR(LOGGER,
+               "IK plugin for group '%s' relies on deprecated API. "
```

Comment:
For some reason kinematics_plugin_loader.cpp is calling this deprecated code in on of its methods. I don't see why at the moment. The reason I added the method is because it the existing KinematicsBase::initialize needed a node as an input. Now that you point it out, I can just remove the node input instead of overloading it.

Reply:
Actually I take that back. KinematicsBase::initialize is a virtual method. I'm guessing it needs a default implementation because it is a plugin. The error message will occur if the user forgets to define the initialize message. So its is saying that their code  is deprecated, not ours.